### ISSUE TITLE
Sass-Lint terminated with error: [object Object]

### ISSUE THREAD

**Issue by karlosdavila** [brackets-userland/brackets-sass-lint#3]
I keep getting this error, even when the Sass/SCSS file is empty.
I also try a new clean Brackets install in a Linux virtual environment, without any other extensions, same results.

Mac OS X 10.11.6 / Xubuntu 16.04
Brackets v1.7

**Comment by petetnt** [comment]
Hi @karlosdavila,

thanks for this report. Could you check out developer tools (`Debug -> Show Developer Tools`) and see if there are any error messages on the `console`.

**Comment by karlosdavila** [comment]
Ok, this is the error message I get:

[CodeInspection] Provider Sass-Lint (async) failed: [object Object]

(anonymous function) @ /language/CodeInspection.js:278

**Comment by petetnt** [comment]
Hmm, the failure occurs on `scanFile`, https://github.com/adobe/brackets/blob/master/src/language/CodeInspection.js#L279, but the extension fails to show the message because the error is an array containing an object unlike those emitted by linting errors.

While I fix that issue, could you open the `brackets-sass-lint/main.js` file from your extensions folder (`Help -> Show Extensions Folder`) and add some  `console.log`s in line 62, so it would look like this:

```
        nodeDomain.exec('lintFile', text, fullPath, fileExt, projectRoot)
            .then(function (results) {
            console.log(results.messages);
            return deferred.resolve({errors: _parseErrorsAndWarnings(results)});
        }, function (err) {
            console.log(err);  
            deferred.reject(err);
        });

```

then reload Brackets (`F5`) and see what the console shows after that.

**Comment by karlosdavila** [comment]
Here is a screenshot of the console results:
[Screen Shot 2016-09-01 at 03.36.31.png](https://dl.dropboxusercontent.com/u/54712/Screen%20Shot%202016-09-01%20at%2003.36.31.png)

**Comment by petetnt** [comment]
Hmm, nothing helpful there either. I'll try to create an version with more helpful error logging tonight and see if I can replicate the issue.
